.github/stale.yml

```diff
@@ -0,0 +1,48 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 14
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - internal
+  - kind/bug
+  - kind/bug-qa
+  - kind/task
+  - kind/feature
+  - kind/design
+  - kind/ci-improvements
+  - kind/performance
+  - kind/flaky-test
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: true
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: true
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: true
+
+# Label to use when marking as stale
+staleLabel: status/stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This repository uses a bot to automatically label issues which have not had any activity (commit/comment/label) 
+  for 60 days. This helps us manage the community issues better. If the issue is still relevant, please add a comment to the 
+  issue so the bot can remove the label and we know it is still valid. If it is no longer relevant (or possibly fixed in the 
+  latest release), the bot will automatically close the issue in 14 days. Thank you for your contributions.
+
+# Limit the number of actions per hour, from 1-30. Default is 30
+limitPerRun: 30
+
+# Limit to only `issues`
+only: issues
```

Comment:
Can we add support for prs? some have been open since 2019.

Reply:
This was explicitly requested by Denise to limit to issues at introduction.